apps/meteor/packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -4002,6 +4002,7 @@
   "Sidebar": "Sidebar",
   "Sidebar_list_mode": "Sidebar Channel List Mode",
   "Sign_in_to_start_talking": "Sign in to start talking",
+  "Signalling_connection_disconnected": "Signalling connection disconnected",
```

Comment:
```suggestion
  "Signaling_connection_disconnected": "Signaling connection disconnected",
```

Reply:
Fixed.

Reply:
looks like you renamed just the key, but not the value 🙈

Reply:
Fixed. :P